support-workers/src/typescript/services/salesforce.ts

```diff
@@ -88,9 +88,15 @@ export class SalesforceService {
 		user: User,
 		giftRecipient: GiftRecipient | null,
 	): Promise<SalesforceContactRecord> => {
-		const buyerResponse = await this.upsert(
-			createContactRecordRequest(user, giftRecipient),
+		console.log('UUU SalesforceService: Creating contact records');
+		console.log('UUU user:', user);
+		const contactRecordRequest = createContactRecordRequest(
+			user,
+			giftRecipient,
 		);
+		console.log('UUU contactRecordRequest:', contactRecordRequest);
+		// First upsert the buyer contact record
+		const buyerResponse = await this.upsert(contactRecordRequest);
```

Comment:
a few log lines and a comment (also makes the diff look more than it is)

```suggestion
		const buyerResponse = await this.upsert(
			createContactRecordRequest(user),
		);
```

Reply:
Yep, removed these. Thanks